ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -115,7 +115,7 @@ Facility\ nurse = Facility nurse
 Failed\ validation = Failed validation
 File\ not\ found = File not found
 For\ example = Eg\:
-Forms = Forms
+app.forms = App forms
```

Comment:
Don't change the translation key - if this key is used elsewhere add a new key and leave the old one unchanged, if it's not used then just change the value and leave the key.